src/reducers/files.js

```diff
@@ -30,16 +30,16 @@ const archive = (state = initialState.get("archive"), action) => {
 
 const collection = (state = List([]), action) => {
   const { id } = action;
-  const collectionEntry = { id };
+  const collectionEntry = Map({ id });
```

Comment:
@shiftsave this is a breaking change. You may need to wipe out your database so it can be re-written